src/classes/REL_Relationships_Con_TDTM.cls

```diff
@@ -41,113 +41,205 @@ public class REL_Relationships_Con_TDTM extends TDTM_Runnable {
-                    } else if (triggerAction == TDTM_Runnable.Action.afterUpdate && relatedContactId != null 
-                            && relatedContactId != oldMap.get(c.id).get(rac.Field__c)){
-                        contactIds.add(c.Id);
-                        reciprocalIds.add(relatedContactId);
-                        types.add(rac.Relationship_Type__c);
-                    }         
-                }
-                
-                //handle updates to the gender field
-                if (triggerAction == TDTM_Runnable.Action.afterUpdate && oldMap.get(c.id) != null) {
-                    if (c.Gender__c != oldMap.get(c.id).Gender__c) {
-                        changedGender.add(c.id);
-                    }
-                }           
-            }
         }
-        
-        if(triggerAction == TDTM_Runnable.Action.AfterDelete) {
-            DmlWrapper w = deleteEmptyRelationships(oldMap);
-            dmlWrapper.objectsToDelete.addAll(w.objectsToDelete);
-        }
-        
-        if (!contactIds.isEmpty()){
-            DmlWrapper w = createNewRelationships(contactIds, reciprocalIds, types);
-            dmlWrapper.objectsToInsert.addAll(w.objectsToInsert);
-        }
-        
-        if (!changedGender.isEmpty()) {
-            //query for a map of all relationships where where contact or related contact has changed gender
-            map<id,Relationship__c> mapRelationship = new map<Id,Relationship__c>(
-                [SELECT Id, Contact__c, RelatedContact__c, ReciprocalRelationship__c, Type__c 
-                FROM Relationship__c WHERE Contact__c IN :changedGender OR RelatedContact__c IN :changedGender]
-            );
-            //make a list of "Related Contact" lookup relationships in our set, these we want to update
-            list<Relationship__c> listRelsForUpdate = new list<Relationship__c>();
-            //make a list of mirror relationships for our first list, we need these for the addType method
-            list<Relationship__c> listMirrorRels = new list<Relationship__c>();
-
-            //get the relationships that the gender change affects
-            for (Relationship__c rel : mapRelationship.values()) {
-                //add the changed gender's relationship and reciprocal
-                if (changedGender.contains(rel.RelatedContact__c)) {
-                    listRelsForUpdate.add(rel);
-                    listMirrorRels.add(mapRelationship.get(rel.ReciprocalRelationship__c));
-                }
-            }
 
-            //update type info
-            REL_Relationships_TDTM.addType(listMirrorRels, listRelsForUpdate);
-            dmlWrapper.objectsToUpdate.addAll((list<sObject>)listRelsForUpdate);
+        UTIL_Describe.cacheDescribeSObjectResult(Contact.SObjectType.getDescribe());
+
+        if (triggerAction == TDTM_Runnable.Action.AfterInsert){
+            return this.handleAfterInsert(newList);
         }
 
+        System.debug('Continuing to old code');
```

Comment:
@m-kinkade saw this while i was working on this class and thought you might want to get rid of this line. :)